### DISCUSSION TITLE
essay/2024/01/12/freedom-completed

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#164] [Comments]
# essay/2024/01/12/freedom-completed

1인 개발자의 좋은 점은 완전한 자유입니다. 회사에 다니던 시절은 대부분 즐거운 시간들이었습니다만.. 다시 돌아가라고 하면 머뭇 거리게 되는 이유는 바로 이것 때문.

https://jeho.page/essay/2024/01/12/freedom-completed.html

**Comment by orangegom** [comment]
항상 글 올라오면 재밌게 잘 보고 있습니다.

읽으면서 저도 어떤 삶을 살고 싶은지 다시 생각하게 되네요 감사합니다.

**Reply by BenjaminKim** [reply]
재밌게 잘 봐주신다니 기쁩니다!

**Comment by happyjy** [comment]
"회사를 그만두고 1인 개발을 해야 할까요?" 글을 보게된 이후로 게시글을 자주 보게 되는데 가족사진이 너무 아름다워 보여요! 
앞으로도 영감을 많이 받는 글을 올려주시면 너무 감사하게 보겠습니다. : )

**Reply by BenjaminKim** [reply]
고맙습니다. 열심히 써보겠습니다. 😁
